sktime/dists_kernels/tests/test_compose_tab_to_panel.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from sktime.dists_kernels.compose_tab_to_panel import AggrDist
+from sktime.dists_kernels.scipy_dist import ScipyDist
+from sktime.utils._testing.panel import make_transformer_problem
+from sktime.utils import all_estimators
+
+PAIRWISE_TRANSFORMERS_TAB = all_estimators(
+    estimator_types="transformer-pairwise-tabular", return_names=False
+)
+
+AGGFUNCS = [
+    np.mean,
+    np.std,
+    np.var,
+    np.sum,
+    np.prod,
+    np.min,
+    np.max,
+    np.argmin,
+    np.argmax,
+    np.any,
+]
+
+
+X1_num_pan = make_transformer_problem(
+    n_instances=4, n_columns=4, n_timepoints=5, random_state=1, return_numpy=True
+)
+X2_num_pan = make_transformer_problem(
+    n_instances=5, n_columns=5, n_timepoints=5, random_state=2, return_numpy=True
+)
+
+X1_list_df = make_transformer_problem(
+    n_instances=4, n_columns=4, n_timepoints=5, random_state=1, return_numpy=False
+)
+X2_list_df = make_transformer_problem(
+    n_instances=5, n_columns=5, n_timepoints=5, random_state=2, return_numpy=False
+)
+
+X1_num_df = np.array(X1_list_df)
+X2_num_df = np.array(X2_list_df)
+
+
+def test_aggr_():
```

Comment:
```suggestion
def test_aggr():
```
Unless there was a reason for the underscore...

Reply:
:joy: good spot